pkg/resources/cruisecontrol/configmap.go

```diff
@@ -37,7 +37,7 @@ func (r *Reconciler) configMap(log logr.Logger) runtime.Object {
     bootstrap.servers=%s:%d
     # The zookeeper connect of the Kafka cluster
     zookeeper.connect=%s/
-`, fmt.Sprintf(kafka.HeadlessServiceTemplate, r.KafkaCluster.Name), r.KafkaCluster.Spec.ListenersConfig.InternalListeners[0].ContainerPort, strings.Join(r.KafkaCluster.Spec.ZKAddresses, ",")) +
+`, fmt.Sprintf(kafka.AllBrokerServiceTemplate, r.KafkaCluster.Name), r.KafkaCluster.Spec.ListenersConfig.InternalListeners[0].ContainerPort, strings.Join(r.KafkaCluster.Spec.ZKAddresses, ",")) +
```

Comment:
what is the motivation behind this?

Reply:
Headless Service cause issues in Istio, especially in Multi Cluster environment, but we need a service which points to all of the brokers for Cruise Control.